frontend/src/Graph/D3Graph.js

```diff
@@ -319,6 +320,13 @@ class D3Graph extends Component {
   render() {
     return (
       <div className='graph-container'>
+        <button
```

Comment:
Why not use the antdesign button component?: 
```
import {Button} from 'antd'
```
It supports className and onClick. And class is the correct className property in React, not class

Reply:
Just do:
```
<Button onClick={this._saveGraph} className="save-graph">
   <Icon type="save" />
</Button>
```

Reply:
Ant design is a component library, not a set of CSS stylesheets ;)